src/app/Main/Header/Export/Export-GenerateLabels.component.tsx

```diff
@@ -20,7 +20,7 @@ const buttonLableStyle = (open: boolean): SerializedStyles => css`
  * @param {boolean} open - true if export dialog is open
  * @returns {[JSX.Element, string, MainFilterLabelInterface, string]} - list of labels: for the export button, for the file, of the mainfilters, for "all values"
  */
-export function generateExportLabels(
+export function ExportGenerateLabelsComponent(

```

Comment:
reject: this actually is no component. Perhaps you should refactor this, so that it only returns the strings. The calling component should then create the actual export button label html code. Then this could be a service again.

Reply:
It still returns not only strings, but the button componente is now separate. commit: "Change how export labels are generated"